.evergreen/run-tests.sh

```diff
@@ -136,6 +136,10 @@ if [ -n "$TEST_ENCRYPTION" ] || [ -n "$TEST_FLE_AZURE_AUTO" ] || [ -n "$TEST_FLE
         exit 1
     fi
     export PYMONGOCRYPT_LIB
+
+    # TODO: Test with 'pip install pymongocrypt'
+    git clone https://github.com/mongodb/libmongocrypt.git libmongocrypt_git
```

Comment:
What I originally had in https://github.com/mongodb/mongo-python-driver/pull/1477 that we removed was:

```
if [ ! -d "libmongocrypt_git" ]; then
	git clone https://github.com/mongodb/libmongocrypt.git libmongocrypt_git
fi
```

Reply:
Do you want me to add that back?

Reply:
Yes please, it errors on subsequent runs otherwise.

Reply:
Done.